addons/link_tracker/tests/test_link_tracker.py

```diff
@@ -154,3 +154,6 @@ def test_no_external_tracking(self):
             link,
             {**expected_utm_params, 'a': 'example.com'}
         )
+
+    def test_no_loop(self):
```

Comment:
docstring for clarity on what exactly we're trying to prevent? Doesn't seem immediately obvious

Reply:
Note that the docstring is added in second commit while the code is in the first :disguised_face: